docker-compose.yml

```diff
@@ -17,6 +17,20 @@ services:
       # - ./docker/db/data:/var/lib/postgresql/data:rw
 ###< doctrine/doctrine-bundle ###
 
+###> symfony/mailer ###
+  web:
+    image: node:16-alpine
+    working_dir: /app
+    command: ["symfony", "server:start"]
+    volumes: [ '.:/app' ]
+###< symfony/mailer ###
```

Comment:
What's this for? 🤔

Reply:
I was thinking how I can implement the `symfony server:start` command. I'll rethink this one